libs/agent-sm/agent/src/test/java/org/opensearch/javaagent/AgentTests.java

```diff
@@ -15,16 +15,21 @@
 import java.security.Policy;
 import java.util.Set;
 
-public class SystemExitInterceptorTests {
+public class AgentTests {
     @SuppressWarnings("removal")
     @BeforeClass
     public static void setUp() {
         AgentPolicy.setPolicy(new Policy() {
-        }, Set.of(), new String[] { "worker.org.gradle.process.internal.worker.GradleWorkerMain" });
+        }, Set.of(), (caller, chain) -> caller.getName().equalsIgnoreCase("worker.org.gradle.process.internal.worker.GradleWorkerMain"));
```

Comment:
This test can be refactored to use AnyCanExit to ensure that tests cover the logic in AnyCanExit.apply